### ISSUE TITLE
The dataset link provided in your code is no longer available, can you update the dataset link?

### ISSUE THREAD

**Issue by pvbvcv** [davidpengucf/SFDAHPE#4]
The dataset link provided in your code is no longer available, can you update the dataset link?

**Comment by davidpengiupui** [comment]
Hi! if you need surreal dataset, please check issue #5

**Comment by gerrydev-liu** [comment]
Hello, authors
download_data(root, "H3D_crop", "H3D_crop.tar", "https://cloud.tsinghua.edu.cn/f/d4e612e44dc04d8eb01f/?dl=1")
download_data(root, "images", "lsp_dataset.zip",  "https://cloud.tsinghua.edu.cn/f/46ea73c89abc46bfb125/?dl=1")
The download links for H3D_crop and LSP datasets are invalid. Can you provide download links for these datasets? Thank you!

**Comment by pvbvcv** [comment]
> Hello, authors download_data(root, "H3D_crop", "H3D_crop.tar", "https://cloud.tsinghua.edu.cn/f/d4e612e44dc04d8eb01f/?dl=1") download_data(root, "images", "lsp_dataset.zip", "https://cloud.tsinghua.edu.cn/f/46ea73c89abc46bfb125/?dl=1") The download links for H3D_crop and LSP datasets are invalid. Can you provide download links for these datasets? Thank you!

Do you have FreiHand and Human3.6M datasets?

**Comment by gerrydev-liu** [comment]
> > Hello, authors download_data(root, "H3D_crop", "H3D_crop.tar", "https://cloud.tsinghua.edu.cn/f/d4e612e44dc04d8eb01f/?dl=1") download_data(root, "images", "lsp_dataset.zip", "https://cloud.tsinghua.edu.cn/f/46ea73c89abc46bfb125/?dl=1") The download links for H3D_crop and LSP datasets are invalid. Can you provide download links for these datasets? Thank you!
> 
> Do you have FreiHand and Human3.6M datasets?

Yes, I have the FreiHand dataset, but I don't have the Human3.6M dataset in image format.
Do you have H3D_crop and LSP datasets?

**Comment by pvbvcv** [comment]
我有H3D数据集以及LSP数据集，但是我现在急需Human3.6M数据集

**Comment by gerrydev-liu** [comment]
> 我有H3D数据集以及LSP数据集，但是我现在急需Human3.6M数据集

我只有human36m的图像数据，jpg的，之前做视频预测用的，所以没有关键点注释文件，你需要的话可以传给你。
另外H3D_crop和LSP可以传我一份吗？ 非常感谢

**Comment by gerrydev-liu** [comment]
> 我有H3D数据集以及LSP数据集，但是我现在急需Human3.6M数据集

请问您找到Human3.6M数据集了吗？

**Comment by pvbvcv** [comment]
这个数据集本身是公开的，但是适用于代码的是经过特殊处理的，这个数据集找到的概率应该很小了，如果非必要的话，建议可以略过这个数据集



发自我的iPhone


------------------ Original ------------------
From: Gerry R ***@***.***&gt;
Date: Mon,Apr 21,2025 3:52 PM
To: davidpengucf/SFDAHPE ***@***.***&gt;
Cc: Baichao Peng ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [davidpengucf/SFDAHPE] The dataset link provided in your code isno longer available, can you update the dataset link? (Issue #4)



    
我有H3D数据集以及LSP数据集，但是我现在急需Human3.6M数据集
  
请问您找到Human3.6M数据集了吗？

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***&gt;
  gerrydev-liu left a comment (davidpengucf/SFDAHPE#4)
  
我有H3D数据集以及LSP数据集，但是我现在急需Human3.6M数据集
  
请问您找到Human3.6M数据集了吗？

**Comment by gerrydev-liu** [comment]
> 这个数据集本身是公开的，但是适用于代码的是经过特殊处理的，这个数据集找到的概率应该很小了，如果非必要的话，建议可以略过这个数据集
> 
> 
> 
> 发自我的iPhone
> […](#)
好的谢谢，我找到了这个NeuralAnnot处理human3.6m的json，和code中用的很像：
https://wenku.csdn.net/doc/5o258fkngs
你可以看看

**Comment by pvbvcv** [comment]
嗯嗯，感谢，不过我已经不再需要了，祝你成功



发自我的iPhone


------------------ Original ------------------
From: Gerry R ***@***.***&gt;
Date: Mon,Apr 21,2025 8:04 PM
To: davidpengucf/SFDAHPE ***@***.***&gt;
Cc: Baichao Peng ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [davidpengucf/SFDAHPE] The dataset link provided in your code isno longer available, can you update the dataset link? (Issue #4)



    
这个数据集本身是公开的，但是适用于代码的是经过特殊处理的，这个数据集找到的概率应该很小了，如果非必要的话，建议可以略过这个数据集
 
发自我的iPhone
 …
 好的谢谢，我找到了这个NeuralAnnot处理human3.6m的json，和code中用的很像：
 https://wenku.csdn.net/doc/5o258fkngs
 你可以看看
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***&gt;
  gerrydev-liu left a comment (davidpengucf/SFDAHPE#4)
  
这个数据集本身是公开的，但是适用于代码的是经过特殊处理的，这个数据集找到的概率应该很小了，如果非必要的话，建议可以略过这个数据集
 
发自我的iPhone
 …
 好的谢谢，我找到了这个NeuralAnnot处理human3.6m的json，和code中用的很像：
 https://wenku.csdn.net/doc/5o258fkngs
 你可以看看

**Comment by gerrydev-liu** [comment]
谢谢>_<
